rules/impl.bzl

```diff
@@ -169,9 +174,10 @@ def process_android_lint_issues(ctx, regenerate):
     # Append the Android manifest file. Lint requires that the input manifest files be named
     # exactly `AndroidManifest.xml`.
     manifest = ctx.file.manifest
-    if manifest and manifest.basename != "AndroidManifest.xml":
```

Comment:
Lint does seem to work when we pass the file as-is - here it is passed as LibManifest.xml and baseline does report issues. Please let me know if there is any case I missed.